src/SFA.DAS.Commitments.Support.SubSite/assets/javascripts/app.js

```diff
@@ -0,0 +1,113 @@
+var sfa = sfa || {};
```

Comment:
What are the chances this js file was copied from another site?

Reply:
Deleted